engine/db/wal/write_ahead_log.hpp

```diff
@@ -58,7 +58,7 @@ class WriteAheadLog {
       RotateFile();
     }
     int64_t next = global_counter_.IncrementAndGet();
-    fprintf(file_, "%lld %d %s\n", next, type, entry.c_str());
+    fprintf(file_, "%ld %d %s\n", next, type, entry.c_str());
```

Comment:
This part is interesting. lld will not warning on macos. ld will not warning on linux. Cannot satisfy both

Reply:
maybe should use `%I64d`